src/Service/BugService.php

```diff
@@ -2,89 +2,25 @@
 
 namespace App\Service;
 
-use App\Entity\Attachment;
-use App\Entity\Bug;
+use App\Entity\UserRequest;
 use App\Form\Model\Search;
 use App\Repository\BugRepository;
-use App\Traits\UserAwareTrait;
-use Doctrine\Common\Collections\Criteria;
-use Doctrine\ORM\EntityManagerInterface;
 use Symfony\Bundle\SecurityBundle\Security;
-use Symfony\Component\HttpFoundation\File\UploadedFile;
 use Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface;
-use Symfony\Component\Uid\Uuid;
 
-class BugService
+class BugService extends UserRequestService
 {
-    use UserAwareTrait;
-
     public function __construct(
-        private readonly EntityManagerInterface $em,
         private readonly BugRepository $repository,
         private readonly AuthorizationCheckerInterface $authorizationChecker,
-        private readonly NotificationService $notificator,
         private readonly Security $security,
     ) {
+        parent::__construct($this->authorizationChecker, $this->security);
```

Comment:
Can't we also abstract repository ?

Reply:
I don't think we can, because in db bugs and features are not in the same table